corehq/apps/users/templates/users/edit_commcare_user.html

```diff
@@ -178,6 +178,9 @@ <h4 class="modal-title">{% blocktrans with couch_user.human_friendly_name as fri
               <p class="alert alert-warning">
                 <i class="fa fa-exclamation-triangle"></i>
                 {% trans "Bad things will happen if you don't read this" %}
+                {% blocktrans %}
+                  It is important you read the entire message below thoroughly before completing this action.
+                {% endblocktrans %}
```

Comment:
i thought you were replacing the "Bad things.." message with this.

Reply:
oops - a1aface3255